app/config.js

```diff
@@ -11,7 +11,7 @@ var debug = require('debug')('dashboard-proxy:config');
 var config = nconf.use('memory')
                   .argv()
                   .env()
-                  .file({ file: 'config.json', format: hjson });
+                  .file({ file: path.join(__dirname, '..', 'config.json'), format: hjson });
```

Comment:
A few lines below this there is the same path computed (`path.join(__dirname, '..', 'config.json')`). May want to save in a var.